rosdep/base.yaml

```diff
@@ -1023,6 +1023,19 @@ exiv2:
       packages: [exiv2]
   rhel: [exiv2]
   ubuntu: [exiv2]
+exiv2-dev:
```

Comment:
Our policy is to have the key name match the Ubuntu name, so this should be `libexiv2-dev`.

Reply:
Okay, renamed